wayfire.ini

```diff
@@ -198,7 +198,7 @@ slot_bl = <super> <shift> KEY_HOME | <super> KEY_KP1
 slot_b = <super> KEY_DOWN | <super> KEY_KP2
 slot_br = <super> <shift> KEY_END | <super> KEY_KP3
 slot_l = <super> KEY_LEFT | <super> KEY_KP4
-slot_c = <super> KEY_F | <super> KEY_KP5
```

Comment:
Hmm, what is the point of `C`/ I think we wanted to make them close to each other.